### ISSUE TITLE
Add 'debug' and 'verbose' options for all operations using ssh.RunSshCmdWithStdin

### ISSUE THREAD

**Issue by Ganeti-Issues-Migrator** [ganeti/ganeti#1058]
Originally reported of Google Code with ID 1002.

```
Currently a couple of Ganeti operations use the ssh.RunSshCmdWithStdin function, but do not expose the 'verbose' and 'debug' flag to the CLI. It would be helpful to fix that in order to make debugging SSH problems easier.
```
Originally added on 2014-12-04 15:04:22 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2015-12-16 10:26:34 +0000 UTC.

Changed State: Started

**Comment by Ganeti-Issues-Migrator** [comment]
```
Fixed in 2.17

commit 19bd15670cd482aa75b9bd0f648c470caf3f7cd0
Author: Helga Velroyen <helgav@google.com>
Date:   Thu Dec 17 11:30:43 2015 +0100

    Propagate --debug/verbose in gnt-node add/remove/modify
    
    This patch adds and propagates the --verbose and --debug
    options of 'gnt-node add/remove/modify' to the calls of
    'ssh_update'. This way the log level can be easily
    controlled and thus debugging SSH errors will become
    easier.
    
    Signed-off-by: Helga Velroyen <helgav@google.com>
    Reviewed-by: Klaus Aehlig <aehlig@google.com>

commit e7d8701b8afc2c5177aa9ce4e1cbba134280d055
Author: Helga Velroyen <helgav@google.com>
Date:   Tue Dec 15 11:38:49 2015 +0100

    Propagate verbose/debug option to ssh_update calls
    
    This patch extends all backend functions that call
    ssh_update by propagating the verbose and debug
    options from the RenewCrypto function downwards.
    
    Signed-off-by: Helga Velroyen <helgav@google.com>
    Reviewed-by: Klaus Aehlig <aehlig@google.com>


```
Originally added on 2016-01-07 15:51:11 +0000 UTC.

Changed State: Fixed
